src/main.cpp

```diff
@@ -269,9 +282,15 @@ class ButtonThread : public OSThread
     {
         DEBUG_MSG("Long press stop!\n");
         longPressTime = 0;
+        if (shutdown_on_long_stop) {
+            playShutdownMelody();
```

Comment:
seems good - though this brings a good point.  We should probably cleanup screen in general and have a separate UX layer class that does screen, buttons sounds.  Someday!